src/Console/Command/CommandHelp.php

```diff
@@ -312,6 +319,39 @@ public static function getDisplayableAllowedValues(FixerOptionInterface $option)
         return $allowed;
     }
 
+    public static function getLatestReleaseVersionFromChangeLog()
+    {
+        static $version = null;
+
+        if (null !== $version) {
+            return $version;
+        }
+
+        $currentMajor = (int) Application::VERSION;
+        $changelogFile = __DIR__.'/../../../CHANGELOG.md';
+
+        if (!is_file($changelogFile)) {
+            return null;
+        }
+
+        $changelog = file_get_contents($changelogFile);
+
+        for ($i = $currentMajor; $i > 0; --$i) {
+            preg_match('/Changelog for v('.$i.'.\d+.\d+)/', $changelog, $matches);
+
+            if (2 === count($matches)) {
```

Comment:
`preg_match()` can be used directly in the `if` so you don't need to count `$matches`.